src/pivotal-ui/components/typography/typography.scss

```diff
@@ -865,6 +865,48 @@ Headings are spaced so their line height and padding are consistent on one or ma
 
 /*doc
 ---
+title: Lined Headings
+name: type_lined
+parent: type
+---
+
+Lined headings jazz up a normal heading with a background divider. It's cool, trust me.
+
+Requires heading text to be wrapped in a nested `<span>`.
+
+Use the `lined-heading` class in conjunction with a background class, e.g. `bg-neutral-11`,
+so the line is blocked out by the text. Also use `txt-c` to make sure text is center-aligned.
+
+**Note**: Lined headings are not meant to be used for multiline text.
+
+```html_example_table
+<h3 class="lined-heading bg-neutral-11 txt-c"><span>One line heading</span></h3>
+```
+
+*/
+
+.lined-heading {
+  position: relative;
+  &:after {
+    display: block;
+    content: " ";
+    border-bottom: 2px solid $divider-alternate-color;
+    position: absolute;
+    width: 100%;
+    bottom: calc(0.5em + 2px);
```

Comment:
@aredridel @jefflembeck Looks like `bottom: 1ex;` gives us the same result. I will change. But should this be a variable?
<img width="442" alt="screen shot 2016-05-12 at 5 24 20 pm" src="https://cloud.githubusercontent.com/assets/1929625/15230960/6ac9e930-1866-11e6-93e2-4e7838943533.png">